integration/feature/init/src/MillInitSbtTests.scala

```diff
@@ -1,16 +1,21 @@
 package mill.integration
 
 import mill.constants.Util
-import mill.integration.MillInitSbtUtils.bumpSbtTo1107
+import mill.integration.MillInitSbtUtils.bumpSbt
 import mill.integration.MillInitUtils.*
 import utest.*
 
 import scala.collection.immutable.SortedSet
 
 object MillInitSbtUtils {
-  def bumpSbtTo1107(workspacePath: os.Path) =
-    // bump sbt version to resolve compatibility issues with lower sbt versions and higher JDK versions
-    os.write.over(workspacePath / "project" / "build.properties", "sbt.version = 1.10.7")
+  /**
+   * bump sbt version to our tested version to resolve compatibility issues with lower sbt versions and higher JDK versions
+   */
+  def bumpSbt(workspacePath: os.Path) =
+    os.write.over(
+      workspacePath / "project" / "build.properties",
+      s"sbt.version = ${sys.props("TEST_SBT_VERSION")}"
```

Comment:
Unfortunately, we need this ugly `getOrElse`, as `sys.props` is returning `null` for undefined properties, making this issue of a missing property hard to diagnose.

```suggestion
      s"sbt.version = ${sys.props.getOrElse("TEST_SBT_VERSION", ???)}"
```